doc/guide/Guide_Miscellaneous.ipynb

```diff
@@ -203,15 +203,199 @@
     "\n",
     "will be performed. Once the change is active, known developers will be notified and this message updated."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Web APIs\n",
+    "CLIMADA relies on open data available through web APIs such as those of the World Bank, Natural Earth, NASA and NOAA.\n",
+    "You might execute the test ``climada_python-x.y.z/test_data_api.py`` to check that all the APIs used are active.\n",
+    "If any is out of service (temporarily or permanently), the test will indicate which one.\n",
+    "\n",
+    "## Manual download\n",
+    "\n",
+    "As indicated in the software and tutorials, other data might need to be downloaded manually by the user. The following table shows these last data sources, their version used, its current availabilty and where they are used within CLIMADA:\n",
+    "\n",
+    "| Availability | Name                                            | Version | Link | CLIMADA class | CLIMADA version | CLIMADA tutorial reference    |\n",
+    "|--------------|-------------------------------------------------|---------|------|---------------|-----------------|-------------------------------|\n",
+    "| OK           | Fire Information for Resource Management System |         |[FIRMS](https://firms.modaps.eosdis.nasa.gov/download/) | BushFire      | >V1.2.5          | climada_hazard_BushFire.ipynb |\n",
+    "| OK           | Gridded Population of the World (GPW)           | v4.11   |[GPW4.11](https://sedac.ciesin.columbia.edu/data/set/gpw-v4-population-count-rev11)     | LitPop        | > v1.2.3        | climada_entity_LitPop.ipynb   |\n",
+    "| FAILED       | Gridded Population of the World (GPW)           | v4.10   |[GPW1.10](https://sedac.ciesin.columbia.edu/data/set/gpw-v4-population-count-rev10)      | LitPop        | >= v1.2.0       | climada_entity_LitPop.ipynb   |\n",
+    "|              |                                                 |         |      |               |                 |                               |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Side Note on Parameters"
```

Comment:
This was a separate notebook, but I think it makes sense to include it here as it isn't that central?

Reply:
Good point!